proto/heimdall/types/divident.proto

```diff
@@ -14,8 +14,7 @@ message DividendAccount {
     option (gogoproto.goproto_getters)  = false;
     option (gogoproto.goproto_stringer) = false;
 
-    string user = 1
-        [(gogoproto.casttype) =
-        "github.com/cosmos/cosmos-sdk/types.AccAddress"];
+    string user       = 1 [(gogoproto.casttype) =
```

Comment:
do we have to remove type cast here as well @murthyvitwit @temaniarpit27

Reply:
Yes, here too.

Reply:
We should remove casttypes to pubkeys, hashes, and addresses.